packages/schematics/src/command-line/nx.ts

```diff
@@ -1,49 +1,125 @@
 #!/usr/bin/env node
-import * as yargsParser from 'yargs-parser';
+import * as yargs from 'yargs';
 
+import * as yargsParser from 'yargs-parser';
 import { affected } from './affected';
 import { format } from './format';
 import { update } from './update';
 import { patchNg } from './patch-ng';
 import { lint } from './lint';
 import { workspaceSchematic } from './workspace-schematic';
-import { generateGraph } from './dep-graph';
+import { generateGraph, OutputType } from './dep-graph';
 
-const processedArgs = yargsParser(process.argv, {
-  alias: {
-    app: ['a']
-  },
-  string: ['app']
-});
-const command = processedArgs._[2];
 const args = process.argv.slice(3);
 
-switch (command) {
-  case 'affected':
-    affected(args);
-    break;
-  case 'dep-graph':
-    generateGraph(yargsParser(args));
-    break;
-  case 'format':
-    format(args);
-    break;
-  case 'migrate': // TODO: delete this after 1.0
-    update(args);
-    break;
-  case 'update':
-    update(args);
-    break;
-  case 'lint':
-    lint();
-    break;
-  case 'postinstall':
-    patchNg();
-    update(['check']);
-    break;
-  case 'workspace-schematic':
-    workspaceSchematic(args);
-    break;
-  default:
-    throw new Error(`Unrecognized command '${command}'`);
-}
+yargs
+  .usage('Usage: $0 <command> [options>')
+  .command(
+    'affected',
+    'Compute affected applications',
+    yargs =>
+      yargs
+        .positional('action', {
+          choices: ['apps', 'build', 'e2e', 'dep-graph']
+        })
+        .command('<action> -- <SHA1 SHA2>', 'Targets within a range of SHAs')
+        .command(
+          '<action> -- files="<file,list>"',
+          'Targets a comma delimited list of files'
+        )
```

Comment:
The files example wraps in my terminal (not sure if that is something you can adjust or not...)
```
Commands:
  nx affected <action> -- <SHA1 SHA2>       Targets within a range of SHAs
  nx affected <action> --                   Targets a comma delimited list of
  files="<file,list>"                       files
  nx affected <action> -- --uncommitted     Targets uncommited changes
  nx affected <action> -- --untracked       Targets untracked changes
  nx affected dep-graph                     Generate a graph showing links
                                            between targets
```